tests/test_filter_params.py

```diff
@@ -27,19 +27,35 @@ def test_filter_params(self, client, entities, model):
         # Setup
         endpoint = ENTITY_ENDPOINT_MAP[model]
         filter_params = ENTITY_PARAMS['filter_params'][endpoint]['valid']
-        expected_total = model.query.filter_by(
-            **filter_params).count()
+        q = model.query
+
+        # List type filter params are handled differently
+        list_params = {}
+        query_params = {}
+        for k, v in filter_params.items():
+            if isinstance(v, list):
+                q = q.filter(getattr(model, k).contains(v))
+                list_params[k] = v
+            else:
+                query_params[k] = v
+
+        # Non-list type filter params
+        expected_total = q.filter_by(**query_params).count()
 
         # Make query string
-        qs = urlencode(filter_params)
+        for k, v in list_params.items():
+            query_params[k] = ','.join(v)
```

Comment:
This is a minor suggestion, but since the iteration of `list_params` is not necessary, 

```python
q = model.query

query_params = {}
expected_total = 0
for k, v in filter_params.items():
    if isinstance(v, list):
        q = q.filter(getattr(model, k).contains(v))
        query_params[k] = ','.join(v)
        expected_total += 1
    else:
        query_params[k] = v

expected_total = q.filter_by(**query_params).count() - expected_total
```

Reply:
Hey @liberaliscomputing so this won't work unfortunately ... let me give you an example:

let's say we're querying Biospcimen and the `filter_params` equals: 

```
{
    'analyte_type': 'DNA',
    'duo_ids': ['DUO:0000021', 'DUO:0000025']
} 
```

then query params will equal:

```
{
    'analyte_type': 'DNA',
    'duo_ids': 'DUO:0000021,DUO:0000025'
} 
```

which will go into the `filter_by` method and result in a SQL query roughly like
this:

```
select * from biospecimen
where analyte_type='DNA' and duo_ids='DUO:0000021,DUO:0000025'
```

Since `duo_ids` column has an ARRAY type that query ^ won't actually get us 
what we want: specimens with analyte_type = DNA and whose duo_ids list
contains both `DUO:0000021`, and `DUO:0000025`